[TAN-5052] Fix scroll position in content builder

AchrafGoVocal opened the pull request:
# Changelog
## Fixed
- Fixed Content Builder scroll position after settings closure

AchrafGoVocal committed:
[TAN-5052] Refactor ContentBuilder to remove portal and simplify rendering

IvaKop commented on front/app/components/admin/ContentBuilder/FullscreenContentBuilder/index.tsx:
```diff
@@ -101,17 +99,9 @@ export const ContentBuilder = ({
       h="100vh"
```
Wondering if we still need all of these props with the new setup? Like the z-index, etc.?

AchrafGoVocal replied:
The z-index is maybe safely removable. 
The bgColor, I don't know what it does honestly so I just didn't touch it.
The data-testid I figured is for testing purposes.
However removing one of the others breaks the report builder.

IvaKop commented on front/app/containers/Admin/index.tsx:
```diff
@@ -124,6 +124,21 @@ const AdminPage = memo<Props>(({ className }) => {
       /admin\/projects\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}(\/(?!projects(?:\/|$))[\w-]+)*/
     ) && !isFoldersPage;
 
+  const isReportBuilderEditorRoute = pathname.match(
```
Good approach, I like this. Makes it very clear 👍

IvaKop reviewed (approved):
Nice work! 🚀 
Very important to run the e2e tests for this one.

AchrafGoVocal commented:
I found an issue with the Homepage builder, the MainHeader is still showing. I am looking into it.

AchrafGoVocal committed:
[TAN-5052] Hide MainHeader in Homepage builder

AchrafGoVocal committed:
[TAN-5052] Clean up by removing zIndex

AchrafGoVocal commented:
I have removed zIndex and removed the constant as well. It seems like it was also used for the Tooltip in the ReportBuilder, removing it however doesn't seem to change anything (e2e tests to confirm).

IvaKop reviewed (commented):
LGTM 🚀
